mjidris/irondb
# US-165 T-194 Create Help views

## Objective
Create the Help views for both authenticated and unauthenticated viewers.

## Detailed Description
I've created two separate components, one for the overall view page, and one that is embedded within the page if the user is authenticated. I also added a `pages` folder within `components` so we can keep track of the pages more easily. We can create more folders for different components as needed.

The two most important files to review to see React code in action are `irondb/client/src/components/pages/Help.jsx`, and `irondb/client/src/components/AuthenticatedHelp.jsx`.

## Screenshot (if applicable)


## How to Test (if applicable)
Similar to #20 :
In order to test this change, navigate to irondb/client, and then run npm start. You can then look at the Help page to see the differences when the user is authenticated or not.

You will see a navigation bar in the top of the application. In order to change from the authenticated to unauthenticated view (or vice-versa), using your text editor, navigate to irondb/client/src/index.js, and on line 16, change the boolean value for isAuthenticated to true or false.
Full authentication integration will take place in a future US.

You can leave the React server running while you change this value, and the app will auto-reload.

## Tests (if added)


## Requirements
[US-165](https://tree.taiga.io/project/cbrobsto-iron-db/us/165)
[T-194](https://tree.taiga.io/project/cbrobsto-iron-db/task/194)
